tests/test_formatted_json.py

```diff
@@ -0,0 +1,298 @@
+import inspect
+import io
+from argparse import Namespace
+
+import pytest
+from flake8.violation import Violation
+
+from flake8_json_reporter.reporters import FormattedJSON
+
+
+def clean(output):
+    return inspect.cleandoc(output) + "\n"
```

Comment:
we shouldn't be "cleaning" the output -- we should be asserting the values *exactly*